src/app/pages/property-details/property-details.component.spec.ts

```diff
@@ -1,78 +1,85 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import {CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
+import {ReactiveFormsModule} from '@angular/forms';
+import { of } from 'rxjs';
+import { ActivatedRoute, Router } from '@angular/router';
+import { NgxSpinnerModule } from 'ngx-spinner';
+import { ToastrService } from 'ngx-toastr';
+import {OwlCarousel} from 'ngx-owl-carousel';
+
 import { PropertyDetailsComponent } from 'src/app/pages/property-details/property-details.component';
 import { PropertyDetailComponent } from 'src/app/pages/property-details/property-detail/property-detail.component';
 import { PropertyDescriptionComponent } from 'src/app/pages/property-details/property-description/property-description.component';
 import { ClientAdminComponent } from 'src/app/pages/property-details/client-admin/client-admin.component';
-import { HttpClientTestingModule } from '@angular/common/http/testing';
 import { PropertyDetailService } from 'src/app/services/property-detail/property-detail.service';
 import {
     resetSpies, propertyDetailSpy, toastServiceSpy, routerSpy
 } from 'src/app/helpers/spies';
-import { of } from 'rxjs';
-import { ActivatedRoute, Router } from '@angular/router';
-import { NgxSpinnerModule } from 'ngx-spinner';
-import { ToastrService } from 'ngx-toastr';
+import {SimilarPropertiesComponent} from 'src/app/pages/property-details/similar-properties/similar-properties.component';
+import {ComponentsModule} from 'src/app/components/components.module';
+
 
 describe('Property detail', () => {
     let component: PropertyDetailsComponent;
     let fixture: ComponentFixture<PropertyDetailsComponent>;
```

Comment:
@wangonya I suggest you declare `let component:` & `let fixture:` as `const` since they are not re-assigned.

Reply:
@wangonya based on the sync we had, `component` & `fixture` should remain declared with `let`.